### ISSUE TITLE
[주제1] Test Set 결과 제출 확인 문의

### ISSUE THREAD

**Issue by JimHeo** [DatathonInfo/H.D.A.I.2021#74]
@DatathonInfo 

안녕하세요, 담당자님.
The Box 팀입니다.

메일을 통해서 Test Set 예측 마스크를 npy 파일로 송부드렸는데 잘 전달이 되었는지 확인이 어려워 문의드립니다.

또한, 메일로 npy 파일을 송부하면서 추가적인 질문도 드렸으나 답변이 없어 이슈에 다시 여쭤보고자 합니다.

처음에 #56 이슈를 보고 png 파일로 송부를 드렸었으나,
`제공해주신 이미지가 블랙으로만 처리가 되어 있는거 같은데요. 0,1로 마스킹 되는 부분만 화이트 처리 부탁드립니다.`
이러한 답변을 받았습니다.
개인적인 의견으로는 기본적으로 png 포맷은 Unsigned 8 bits를 사용하기 때문에 0~255 사이의 값 중에서 0과 1로만 이루어져 있다면 이미지가 검정색으로만 보이는 것은 당연하다고 생각이 드는데요.
그렇기 때문에, 1의 값을 화이트로 보이게 하기 위해서는 255로 값을 변환시켜줘야 할 것 같은데 #61 이슈를 보면 픽셀은 0과 1로만 구성하는 것이 맞는 듯 합니다. (또한, 기존 Training Set이나 Validation Set에서도 정답 데이터를 uint8로 된 0과 1값의 npy 파일로 주셨던 것으로 알고 있습니다.)
이러한 사항으로 인해서 해당 답변이 잘 이해가 되지 않았고, 우선은 npy 파일로 다시 송부를 드렸습니다.

혹시 제가 알지 못하는 다른 사항이 있다면 이 부분도 답변을 같이 부탁드립니다.

더불어, 대회 기간 도중에 Test Set에 대한 결과를 직접 송부하는 방식으로 변경되었는데 이에 대한 공지를 주실 때 구체적인 파일 포맷이나 혹은 대회 측에서 어떤 라이브러리를 사용하여 파일을 오픈하는지 정도는 같이 고지해 주셨더라면 조금 더 혼선이 없지 않을까 하는 아쉬움이 있네요.

글이 조금 길어졌는데 우선적으로 파일이 문제없이 제출되었는지 알고 싶습니다.
감사합니다.

**Comment by JimHeo** [comment]
@DatathonInfo 혹시 답변은 언제쯤 가능한건가요.....?
